man/smartdc/man/man1/pkgsrc-setup.1.md

```diff
@@ -0,0 +1,25 @@
+# pkgsrc-setup(1) -- bootstrap pkgsrc-tools
+
+
+## SYNOPSIS
+
+`pkgsrc-setup [--clean]`
+
```

Comment:
There's no mention of /path/to/root as an option.  I take it that's deliberate and not an oversight?

Reply:
Added a man page

Reply:
Correct. The path to root is only intended to be used during smartos install when the zpool is mounted to an alternate location. If you just tried to install to an arbitrary location I don't even know if the installation would be usable. On a running system, any location other than `/opt/tools` is not supported.